ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -275,7 +275,10 @@ function configure_terraform() {
     export POWERVS_SERVICE_INSTANCE_ID
 
     echo "Release Image used is:"
-    echo "$(openshift-install version | grep 'release image' | awk '{print $3}')"
+    curl -o /tmp/versions.json -s 'https://multi.ocp.releases.ci.openshift.org/graph?arch=ppc64le'
+    jq -r --arg nightly nightly --arg version ${OCP_VERSION} '[.nodes[] | select(.version | (contains($nightly) and startswith($version)))][0].payload' /tmp/versions.json > /tmp/target_version
+    export TARGET_VERSION="$(< /tmp/target_version)"
```

Comment:
This need to be assigned the variable and then export as its not able to merge the code .

TARGET_VERSION="$(< /tmp/target_version)"
export TARGET_VERSION

because of rehearse job failure - https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58640/pull-ci-openshift-release-master-step-registry-shellcheck/1854884937853833216